packages/fxa-react/configs/tailwind.js

```diff
@@ -62,10 +64,11 @@ module.exports = {
         '-3': '-0.75rem',
         '-50': '-11.50rem',
         '-52': '-13rem',
-        '3': '0.75rem',
-        '6': '1.25rem',
-        '10': '2.5rem',
-        '55': '13.75rem',
+        0.5: '0.125rem',
+        3: '0.75rem',
+        6: '1.25rem',
+        10: '2.5rem',
+        55: '13.75rem',
```

Comment:
Prettier decided to make a few tweaks to sections of these files. Can investigate if anything looks incorrect.